### PR TITLE
 	Standardize using <br /> instead of variations

### PR DESCRIPTION
**Pull request by panpawn**

### CODE REVIEW

**Comment by ascriptmaster** [comment]
Is there anywhere where we use `<br />` in existing code? This isn't XML; the end slash is hardly necessary for this particular tab, so if we don't use it anywhere already I don't think it needs a change

Standardizing is nice, though, I guess.

**Comment by panpawn** [comment]
>Is there anywhere where we use <br /> in existing code? 

Yes, in most cases where we use a br tag, we use it as ``<br />``.

**Comment by panpawn** [comment]
@Slayer95 - would you prefer a different standardization?

**Comment by Zarel** [comment]
I overall prefer `<br />` and consistently use it.

This isn't XHTML but `<br />` is intentionally allowed in HTML5, and I think `<br />` makes for more readable code by making it clear which tags autoclose and which tags don't.

**Comment by Zarel** [comment]
I'd wait for @Slayer95's reply, but since the code is currently unstandardized, this is worth merging. We can convert to a different standard if you can convince me of it.

File: chat-plugins/datasearch.js [hunk 1]
```diff
@@ -669,7 +669,7 @@ function runDexsearch(target, cmd, canAll, message) {
 		results = Tools.shuffle(results).slice(0, randomOutput);
 	}
 
-	let resultsStr = (message === "" ? message : "<font color=#999999>" + escapeHTML(message) + ":</font><br>");
+	let resultsStr = (message === "" ? message : "<font color=#999999>" + escapeHTML(message) + ":</font><br />");
```
Comment (kotarou3) [inline chat-plugins/datasearch.js]: It would also be nice if we could get rid of the use of obsolete tags like `<font>`

Comment (panpawn) [inline chat-plugins/datasearch.js]: Yes I agree, but that is a separate issue :P

**Pull request merged**